src/serializers/html.js

```diff
@@ -108,10 +109,21 @@ class Html {
         return memo
       }
 
-      const block = {
-        kind: 'block',
-        type: this.defaultBlockType,
-        nodes: [node]
+      let block
+      if (this.defaultBlockType.constructor.name === 'String') {
+        block = {
+          kind: 'block',
+          type: this.defaultBlockType,
+          nodes: [node]
+        }
+      } else {
+        block = Object.assign(
```

Comment:
Can you change this to use the object spread syntax (`{ ...this.defaultBlockType }`) instead? Thanks